webapp/app.py

```diff
@@ -522,7 +522,7 @@ def utility_processor():
 # blog section
 
 blog_views = BlogViews(
-    api=BlogAPI(session=session),
+    api=BlogAPI(session=session, thumbnail_width=555, thumbnail_height=311),
```

Comment:
where do these numbers come from? in my limited testing I only see images at width 333

Reply:
It's from the breakpoint of 620 which is when the UI switches to stacked which results in the image rendering at 555px.

![image](https://user-images.githubusercontent.com/1413534/187464636-470d42f6-b664-4ca9-b354-8d6e78f17a61.png)